client/components/domains/register-domain-step/style.scss

```diff
@@ -153,6 +153,10 @@
 	}
 }
 
+.register-domain-step__placeholder.empty-content {
+	padding-bottom: 30px;
```

Comment:
This evens out the whitespace in the content area above and below the new illustration.